lib/bloc/account/account_bloc.dart

```diff
@@ -274,14 +277,14 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
 
   Future<sdk.ReceivePaymentResponse> addInvoice({
     String description = "",
-    required int amountSats,
+    required int amountMsat,
     required sdk.OpeningFeeParams? chosenFeeParams,
   }) async {
-    _log.info("addInvoice: $description, $amountSats");
+    _log.info("addInvoice: $description, $amountMsat");
 
-    final requestData = sdk.ReceivePaymentRequest(
-        amountSats: amountSats, description: description, openingFeeParams: chosenFeeParams);
-    final responseData = await _breezLib.receivePayment(reqData: requestData);
+    final req = sdk.ReceivePaymentRequest(
+        amountMsat: amountMsat, description: description, openingFeeParams: chosenFeeParams);
+    final responseData = await _breezLib.receivePayment(req: req);
```

Comment:
Can we have `sendSpontaneousPayment`, `sendPayment`, `lnurlPay`, `lnurlWithdraw` to be in this format where the request variable is initialized first then passed to the function?
It would help with debugging & readability.

Reply:
Good point, done